iron-collapse.html

```diff
@@ -278,19 +283,13 @@
       this.toggleClass('iron-collapse-opened', this.opened);
       this._updateTransition(false);
       this.notifyResize();
+      this._setTransitioning(false);
     },
 
-    /**
-     * Simplistic heuristic to detect if element has a parent with display: none
-     *
-     * @private
-     */
-    get _isDisplayed() {
```

Comment:
Yes it's safe; this was introduced in https://github.com/PolymerElements/iron-collapse/pull/49 to handle cases where an animation wouldn't be triggered. But this is already handled in `updateSize` by `willAnimate`.